src/components/Message.vue

```diff
@@ -191,6 +188,32 @@ export default {
 		sendReply(data) {
 			return sendMessage(data.account, data)
 		},
+		replyMessage() {
+			this.$router.push({
+				name: 'message',
+				params: {
+					accountId: this.$route.params.accountId,
+					folderId: this.$route.params.folderId,
+					messageUid: 'reply',
+				},
+				query: {
+					uid: this.message.uid,
+				},
+			})
+		},
+		replyAllMessage() {
```

Comment:
you could rename the method and add the `messageUid` as parameter :wink:

Reply:
Do you mean doing something like this?

```
<ActionButton icon="icon-mail" @click="reactOnMessage('reply')">{{ t('mail', 'Reply') }}</ActionButton>
<ActionButton icon="icon-delete" @click="reactOnMessage('replyAll')">{{ t('mail', 'Reply all') }}</ActionButton>
<ActionButton icon="icon-delete" @click="reactOnMessage('forward')">{{ t('mail', 'Forward') }}</ActionButton>
...
 reactOnMessage(action) {
                        this.$router.push({
                                name: 'message',
                                params: {
                                        accountId: this.$route.params.accountId,
                                        folderId: this.$route.params.folderId,
                                        messageUid: action,
                                },
                                query: {
                                        uid: this.message.uid,
                                },
                        })
}
```

I've tried but can't get it to work for some reason; As I said already, I'm a js noob :-).

Reply:
Okay, strange. I expected that to just work :man_shrugging: